pallets/api/src/fungibles/mod.rs

```diff
@@ -386,6 +383,11 @@ pub mod pallet {
 
 		/// Start the process of destroying a token.
 		///
+		/// See `pallet-assets` documentation for more information. Related dispatchables are:
```

Comment:
```suggestion
		// See `pallet-assets` documentation for more information. Related dispatchables are:
```

This and the following lines should not be a doc comment. I'm fairly certain these doc comments for dispatchables goes into the metadata and gets shown in the PJS apps interface, and dont think we want that shown there personally. I normal comment is fine.